netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -340,7 +340,7 @@ private void createStream(CreateStreamCommand command, final ChannelPromise prom
       if (!connection().goAwaySent()) {
         logger.fine("Stream IDs have been exhausted for this connection. "
                 + "Initiating graceful shutdown of the connection.");
-        super.close(ctx(), ctx().newPromise());
+        close(ctx(), ctx().newPromise());
```

Comment:
Sort of. It used to not matter because close() and super.close() behaved basically the same (just logging difference). When I fixed close() to call transportShutdown() I failed to fix this case at the same time.